.github/workflows/nightly_tests.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     runs-on: [single-gpu, nvidia-gpu, t4, ci]
     container:
       image: diffusers/diffusers-pytorch-cuda
-      options: --shm-size "16gb" --ipc host -v /mnt/hf_cache:/mnt/cache/ --gpus 0
+      options: --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface/diffusers:/mnt/cache/ --gpus 0 --privileged
```

Comment:
Can we do `gpus all` here? 

Also, does it make sense to have the cache paths defined as env vars?

Reply:
The cache path is passed to the run options of the container, while the env variables are accessed inside the container. 

Don't think we can just pass the env variable to this command here.